lib/install/management/appliance.go

```diff
@@ -605,6 +606,57 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 	},
 	)
 
+	// Kubelet
+	if conf.KubernetesServiceURL != "" && conf.KubeletConfigFile != "" {
+		vmName, err := vm2.Name(d.op)
+		if err != nil {
+			d.op.Errorf("Failed to get VM name, error: %s", err)
+			return err
+		}
+		kubeletName := fmt.Sprintf("kubelet-%s", vmName)
+		kubeletStarter := executor.Cmd{
+			Path: "/sbin/kubelet-starter",
+			Args: []string{
+				"/sbin/kubelet-starter",
+			},
+		}
+
+		kubeletStarter.Env = append(kubeletStarter.Env, fmt.Sprintf("%s=%s", "KUBELET_NAME", kubeletName))
+
+		plPort := portLayerPort
+		if conf.Diagnostics.DebugLevel > 2 {
+			// expose portlayer service on client interface
+			plPort = constants.DebugPortLayerPort
+		}
+
+		// Set up the persona and port layer
+		kubeletStarter.Env = append(kubeletStarter.Env, fmt.Sprintf("%s=%s:%s", "PERSONA_URL", "localhost", d.DockerPort))
+		kubeletStarter.Env = append(kubeletStarter.Env, fmt.Sprintf("%s=%s:%s", "PORTLAYER_URL", "localhost", plPort))
+
+		if settings.HTTPProxy != nil {
```

Comment:
The CLI help and [documentation](https://vmware.github.io/vic-product/assets/files/html/1.3/vic_vsphere_admin/vch_proxy.html#http-proxy) for these proxy settings suggests that they're only used for pulling images. Is that the only thing that the environment variable being set here is used for? If not, do we need to adjust the CLI help (in `cmd/vic-machine/common/proxy.go`) and documentation?